src/com/owncloud/android/ui/adapter/DiskLruImageCacheFileProvider.java

```diff
@@ -0,0 +1,121 @@
+/**
+ *   ownCloud Android client application
+ *
+ *   Copyright (C) 2015 Tobias Kaminsky
```

Comment:
Please don't add your Copyright in the license. You can put your name as @author